src/Core/Service/Command/Install.php

```diff
@@ -32,6 +35,12 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $io->title('Installing services...');
 
+        if ($this->manager->isDisabled()) {
```

Comment:
If you can think of more places where we should do this check and early return, feel free to add accordingly :)